addons/l10n_ch/models/account_invoice.py

```diff
@@ -29,6 +28,11 @@ class AccountMove(models.Model):
     l10n_ch_currency_name = fields.Char(related='currency_id.name', readonly=True, string="Currency Name", help="The name of this invoice's currency") #This field is used in the "invisible" condition field of the 'Print ISR' button.
     l10n_ch_isr_needs_fixing = fields.Boolean(compute="_compute_l10n_ch_isr_needs_fixing", help="Used to show a warning banner when the vendor bill needs a correct ISR payment reference. ")
 
+    l10n_ch_is_qr_valid = fields.Boolean(compute='_compute_qr_is_valid', help="Determines whether an invoice can be printed as a QR or not")
+
+    def _compute_qr_is_valid(self):
+        self.l10n_ch_is_qr_valid = self.partner_bank_id._qr_is_possible('ch_qr', self.partner_id, self.currency_id)
```

Comment:
A compute must always be usable on multiple records:
```py
for move in self:
    move.l10n_ch_is_qr_valid = move.partner_bank_id._qr_is_possible('ch_qr', move.partner_id, move.currency_id)
```
Also, you need to compute that only on invoices